awx/ui/client/legacy/styles/bootstrap-custom-overrides.less

```diff
@@ -0,0 +1,188 @@
+body {
+    font-family: 'Open Sans', sans-serif;
+    font-weight: 400;
+    color: #161B1F;
+    background-color: #FCFCFC;
+    font-size: 0.88rem;
+}
+
+.dropdown-toggle::after {
+    display:none;
+}
+
+.tooltip.show {
+    opacity: 1;
+}
+
+.tooltip {
+    font-family: 'Open Sans', sans-serif;
+    font-size: 0.75rem;
+    line-height: 1.4;
+}
+
+.bs-tooltip-top .arrow::before,
+.bs-tooltip-auto[x-placement^="top"] .arrow::before {
+    border-top-color: #707070;
+}
+
+.bs-tooltip-bottom .arrow::before,
+.bs-tooltip-auto[x-placement^="bottom"] .arrow::before {
+    border-bottom-color: #707070;
+}
+
+.bs-tooltip-left .arrow::before,
+.bs-tooltip-auto[x-placement^="left"] .arrow::before {
+    border-left-color: #707070;
+}
+
+.bs-tooltip-right .arrow::before,
+.bs-tooltip-auto[x-placement^="right"] .arrow::before {
+    border-right-color: #707070;
+}
+
+/* this used to exist but was removed */
+.form-horizontal .form-group {
+    margin-left: -15px;
+    margin-right: -15px;
+}
+
+label {
+    margin-bottom: 0.3125rem;
+}
+
+.h1, .h2, .h3, .h4, .h5, .h6, h1, h2, h3, h4, h5, h6 {
+    line-height: 1.1;
+}
+
+p {
+    margin-bottom: 0.625rem;
+}
+
+.dropdown-menu, .btn {
+    font-size: 0.88rem;
+}
+
+.btn-xs, .btn-group-xs>.btn {
+    padding: 1px 5px;
+    font-size: 0.75rem;
+    line-height: 1.5;
+    border-radius: 3px;
+}
+
+.btn-sm, .btn-group-sm>.btn {
+    padding: 5px 10px;
+    font-size: 0.75rem;
+    line-height: 1.5;
+    border-radius: 3px;
+}
+
+button, html input[type=button], input[type=reset], input[type=submit] {
+    cursor: pointer;
+}
+
+.dropdown-menu {
+    padding: .2rem 0;
+}
+
+.dropdown-item {
+    padding: 0.19rem 0.625rem;
+}
+
+@media (min-width: 768px) {
+    .modal-dialog {
+        width: 600px;
+    }
+}
+
+.modal-dialog {
+    max-width: none;
+}
+
+.modal-footer>:not(:last-child) {
+    margin-right: 1.25rem;
+}
+
+.btn-group-vertical>.btn, .btn-group>.btn {
+    -ms-flex: 1 1 auto;
+    flex: 1 1 auto;
+}
+
+.bs-popover-auto[x-placement^=right] .arrow::after, .bs-popover-right .arrow::after {
+    border-right-color: #707070;
+}
+
+.bs-popover-auto[x-placement^=left] .arrow::after, .bs-popover-left .arrow::after {
+    border-left-color: #707070;
+}
+
+.bs-popover-auto[x-placement^=top] .arrow::after, .bs-popover-top .arrow::after {
+    border-top-color: #707070;
+}
+
+.bs-popover-auto[x-placement^=bottom] .arrow::after, .bs-popover-bottom .arrow::after {
+    border-bottom-color: #707070;
+}
+
+a:not([href]):not([tabindex]) {
+    color: #337AB7;
+}
+
+a:not([href]):not([tabindex]):hover {
+    color: #286090;
+}
+
+.dropdown-menu>li>a {
+    padding: 3px 10px;
+    display: block;
+    clear: both;
+    font-weight: 400;
+    line-height: 1.42857143;
+    white-space: nowrap;
+}
+
+.dropdown-item.active,
+.dropdown-item:active,
+.dropdown-menu>li>a:hover,
+.dropdown-menu>li>a:focus {
+    text-decoration: none;
+    color: #262626;
+    background-color: #f5f5f5;
```

Comment:
hex values could be color vars in this file

Reply:
👍